### PR TITLE
Added FAQ about discounts for non-profits

### PR DESCRIPTION
**Pull request by JeffSchering**
Resolves issue [Add non-profit discounting FAQ #1039](https://github.com/mattermost/docs/issues/1039)

Also, my text editor removed trailing whitespace.

### CODE REVIEW

File: source/overview/faq.rst [hunk 1]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
+If you need encrypted transport or production-level SLAs for push notifications, you can either compile your own push-proxy service, with your own key, or you can use a paid option and become a Mattermost Enterprise Edition E10 subscriber `agreeing to our Conditions of Use <https://about.mattermost.com/default-terms/>`_, which enables you to use a production-level Hosted Push Notification Service (HPNS) at ``https://push.mattermost.com``, offering transport-level encryption.
 
-Learn more about `our mobile apps and push notification service <https://docs.mattermost.com/deployment/push.html>`_. 
+Learn more about `our mobile apps and push notification service <https://docs.mattermost.com/deployment/push.html>`_.
 
 
-Video, Audio and Screensharing 
+Video, Audio and Screensharing
```
Comment (lindy65) [inline source/overview/faq.rst]: "Screensharing" > "Screen sharing" or "Screen-sharing"

File: source/overview/faq.rst [hunk 2]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
 ----------------------------------------
 
-What support is available for video and audio calling and screensharing?  
+What support is available for video and audio calling and screensharing?
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

File: source/overview/faq.rst [hunk 3]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
 
-In all editions, open source `one-on-one video and audio calling is currently available in beta <https://docs.mattermost.com/deployment/webrtc.html>`_ using the WebRTC standard.  
+In all editions, open source `one-on-one video and audio calling is currently available in beta <https://docs.mattermost.com/deployment/webrtc.html>`_ using the WebRTC standard.
 
-After many conversations with users and customers we realized: 
+After many conversations with users and customers we realized:
 
-1. Most people using Mattermost need top quality video/audio/screensharing solutions, which are predominantly propreitary.   
-2. Integrating an organization's choice of video/audio/screensharing solution is paramount. 
+1. Most people using Mattermost need top quality video/audio/screensharing solutions, which are predominantly propreitary.
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

"propreitary" > "proprietary"

File: source/overview/faq.rst [hunk 4]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
+2. Integrating an organization's choice of video/audio/screensharing solution is paramount.
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

File: source/overview/faq.rst [hunk 5]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
 
-Today, you can neatly link video/audio/screensharing solutions, like Zoom and Skype for Business, to channels by using markdown formatting the header (Example: ``[Click for video call](https://link_to_solution)``). 
+Today, you can neatly link video/audio/screensharing solutions, like Zoom and Skype for Business, to channels by using markdown formatting the header (Example: ``[Click for video call](https://link_to_solution)``).
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

File: source/overview/faq.rst [hunk 6]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
 
-In future, we plan to add: 
+In future, we plan to add:
 
-- A plug-in option for user profiles to connect directly to a person using 3rd party video/audio/screensharing products, including custom and phone-based solutions. 
-- A plug-in option for linking channels and teams to on-demand video/audio/screensharing solutions 
-- A default, open source one-on-one video and audio calling plug-in based on WebRTC 
-- A directory of 3rd party video/audio/screensharing solutions that plug-in to Mattermost 
+- A plug-in option for user profiles to connect directly to a person using 3rd party video/audio/screensharing products, including custom and phone-based solutions.
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

File: source/overview/faq.rst [hunk 7]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
+- A plug-in option for linking channels and teams to on-demand video/audio/screensharing solutions
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

File: source/overview/faq.rst [hunk 8]
```diff
@@ -12,47 +12,47 @@ Why was Mattermost created?
...
+- A default, open source one-on-one video and audio calling plug-in based on WebRTC
+- A directory of 3rd party video/audio/screensharing solutions that plug-in to Mattermost
```
Comment (lindy65) [inline source/overview/faq.rst]: "screensharing" > "screen sharing" or "screen-sharing"

File: source/overview/faq.rst [hunk 9]
```diff
@@ -65,92 +65,98 @@ What is Mattermost Enterprise Edition?
...
 
-Other considerations: 
+Other considerations:
 
-- Mattermost has a default `Conditions of Use <https://docs.mattermost.com/administration/config-settings.html#terms-of-service-link>`_ agreement for the Terms of Service link at the bottom of login screen that should be incorporated into any additional Terms of Use you may add. 
-- The Mattermost copyright notices on the user interface should remain. 
-- There may be additional legal and regulatory issues to consider and we recommend you employ legal council to fully understand what's involved in creating and selling a derivative work. 
+- Mattermost has a default `Conditions of Use <https://docs.mattermost.com/administration/config-settings.html#terms-of-service-link>`_ agreement for the Terms of Service link at the bottom of login screen that should be incorporated into any additional Terms of Use you may add.
+- The Mattermost copyright notices on the user interface should remain.
+- There may be additional legal and regulatory issues to consider and we recommend you employ legal council to fully understand what's involved in creating and selling a derivative work.
```
Comment (lindy65) [inline source/overview/faq.rst]: "council" > "counsel"

Comment (JeffSchering) [inline source/overview/faq.rst]: Nice! Good catch.

**Pull request merged**